cmd/software/deployment.go

```diff
@@ -143,7 +143,11 @@ func newDeploymentCreateCmd(out io.Writer) *cobra.Command {
 	}
 
 	if triggererEnabled {
-		cmd.Flags().IntVarP(&createTriggererReplicas, "triggerer-replicas", "", 1, "Number of replicas to use for triggerer airflow component, valid 0-2")
+		cmd.Flags().IntVarP(&createTriggererReplicas, "triggerer-replicas", "", 0, "Number of replicas to use for triggerer airflow component, valid 0-2")
+	}
+
+	if !cmd.Flags().Changed("triggerer-replicas") {
+		createTriggererReplicas = -1
 	}
```

Comment:
This logic needs to be in `deploymentCreate` function in this file, over here `cmd.Flags().Changed("triggerer-replicas")` will always evaluate to `false`